pkg/machines/components/consoleClientResources.jsx

```diff
@@ -0,0 +1,58 @@
+/*jshint esversion: 6 */
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+import React from "react";
+import cockpit from 'cockpit';
+
+const _ = cockpit.gettext;
+
+const ConnectWithSPICEInstructions = ({ displays }) => {
+    if (!displays.spice) {
+        return null
+    }
+    return (
+        <p>
+            {_("For better experience with SPICE, install")}&nbsp;
+            <a href='http://www.spice-space.org/download.html' target='_blank'>{_("SPICE drivers.")}</a>
```

Comment:
Translatable strings must not be split that way, as word order is often different in other languages. Use something like
```js
cockpit.format(_("For better experience with SPICE, install $0 drivers."),
               "<a href='http://www.spice-space.org/download.html' target='_blank'>SPICE</a>");
```